Sources/Prometheus/Prometheus.swift

```diff
@@ -16,10 +16,15 @@ public class PrometheusClient {
     /// Lock used for thread safety
     private let lock: Lock
     
+    /// Sanitizers used to clean up label values provided through
+    /// swift-metrics.
+    public let sanitizers: [LabelSanitizer]
+    
     /// Create a PrometheusClient instance
-    public init() {
+    public init(_ sanitizers: [LabelSanitizer] = [PrometheusLabelSanitizer()]) {
```

Comment:
Are we sure we need multiple sanitizers really?

Reply:
Good point probably. Will update to just one

Reply:
Cool 👍  (Minor nitpick, perhaps worth keeping the argument label?)